decoder/stats.go

```diff
@@ -508,9 +527,16 @@ func logPokemonCount(statsDb *sqlx.DB) {
 					addRows(&nundoRows, pokemonId, count)
 				}
 			}
-			for pokemonId, count := range stats.shiny {
-				if count > 0 {
-					addRows(&shinyRows, pokemonId, count)
+			for pokemonId, checks := range stats.shinyChecks {
+				if checks.total > 0 {
```

Comment:
I guess we should only write if we found a shiny, not if we saw the mon in general, right?

Reply:
It has to be written even if no shiny. There could have been a shiny on previous (or later) runs and accurate total is needed.

Reply:
yes to get an accurate on shiny stats, you need shiny count / by total count, else the data in stats can't be used to calculate a shiny rate seen on that pokemon